### DISCUSSION TITLE
Can we translate texts in frontend/src/static/js/utils/settings/pages.js directly?

### DISCUSSION THREAD

**Discussion by 0000sir** [mediacms-io/mediacms#306] [General]
I was trying to create a localized deployment, and got problem with how to do it.
I change ```PageStore.get('config-enabled').pages.featured.title``` to ```t(PageStore.get('config-enabled').pages.featured.title)``` and it works, but it's not a good practice since I have to make this change every where.

I found text was write in frontend/src/static/js/utils/settings/pages.js and other files, but I'm not good at js or react, I tried a couple of hours to import some thing like i18next or react-i18next, no luck with it.

So is there any way to translate these config file directly or I am in totally wrong way?

If it's possible, how to make it work, thank you for your great advise.

**Comment by mgogoulos** [comment]
there has been an effort to provide full translation but has hang for a number of reasons - there are open PRs btw. This work will soon be continued however as it's an important feature
